firebase-perf/src/main/java/com/google/firebase/perf/metrics/AppStartTrace.java

```diff
@@ -227,11 +272,24 @@ public synchronized void onActivityResumed(Activity activity) {
 
     // Log the app start trace in a non-main thread.
     executorService.execute(this::logAppStartTrace);
+  }
 
-    if (isRegisteredForLifecycleCallbacks) {
-      // After AppStart trace is logged, we can unregister this callback.
-      unregisterActivityLifecycleCallbacks();
```

Comment:
Are we still unregistering somewhere if the the experiment is disabled and recordFirstDrawDone never gets called?

Reply:
Great catch! Added it back.